src/network/clientpackethandler.cpp

```diff
@@ -1302,6 +1302,7 @@ void Client::handleCommand_HudSetSky(NetworkPacket* pkt)
 				>> skybox.sky_color.night_sky >> skybox.sky_color.night_horizon
 				>> skybox.sky_color.indoors;
 		}
+		*pkt >> skybox.shadow_intensity;
```

Comment:
Packet length check is necessary to not trigger a `PacketError` exception.

Reply:
Added a length check and tested with a 5.4 server. Please take a look.